src/api/util.js

```diff
@@ -27,17 +27,31 @@ function isNumber(value) {
     return typeof value == 'number';
 }
 
+function isDate(value) {
```

Comment:
We are not using this function anywhere. I do not know if it was intended to be used [here](https://github.com/CartoDB/carto-vl/pull/550/files#diff-603cdd3c5b0b5e1002450b2c4e338129R40), but if it is not going to be used I would remove it.